manifests/1.14/deploy/vsphere-csi-controller-ss.yaml

```diff
@@ -60,6 +60,18 @@ spec:
               readOnly: true
             - mountPath: /var/lib/csi/sockets/pluginproxy/
               name: socket-dir
+          ports:
+            - name: healthz
+              containerPort: 9808
+              protocol: TCP
+          livenessProbe:
```

Comment:
I had to go read the [docs](https://github.com/kubernetes-csi/livenessprobe#usage) to remind myself why `livenessProbe` was set here rather than in the `liveness-probe` container itself.

/lgtm